### PR TITLE
(HC-54) More fixes for tests

### PR DESCRIPTION
**Pull request by MikaelSmith**

### CODE REVIEW

File: lib/tests/conf_parser_test.cc [hunk 1]
```diff
@@ -4,27 +4,133 @@
...
+}
+
+TEST_CASE("valid conf works") {
+    // all we're checking here unfortunately is that it doesn't throw.
+    // for a more thorough check, use the EquivalentsTest stuff.
+    for (auto const& valid : whitespace_variations(valid_conf(), true)) {
+        CAPTURE(valid.test);
+        REQUIRE_NOTHROW(parse(valid.test));
+        //auto our_ast = parse(valid.test);
+        // TODO: implement rendering
```
Comment (MikaelSmith) [inline lib/tests/conf_parser_test.cc]: Need to file a ticket for this.

**Comment by branan** [comment]
a50946f seems basically empty. Did this accidentally get carried along from a rebase or something?

**Comment by MikaelSmith** [comment]
Oh, probably. I'll fix that.

File: lib/tests/test_utils.cc [hunk 2]
```diff
@@ -150,7 +150,7 @@ namespace hocon {
     shared_config parse_config(std::string s) {
         auto options = config_parse_options().
             set_origin_description(make_shared<string>("test string")).set_syntax(config_syntax::CONF);
-        return config::parse_string(move(s), make_shared<config_parse_options>(move(options)));
+        return config::parse_string(move(s), options);
```
Comment (branan) [inline lib/tests/test_utils.cc]: Should we still be moving here?

Comment (branan) [inline lib/tests/test_utils.cc]: Oh, looks like this line got changed again in a later commit. Yay.

**Comment by branan** [comment]
Travis didn't kick, but the only change was removing an empty commit. I'm merging anyway!

**Pull request merged**